### PR TITLE
Add autoloader directives

### LINKED ISSUE #50: Add PSR-4 autoload directives
This would help when autoloading (e.g. with sspak).

https://github.com/silverstripe/sspak/issues/76

I noticed some of the namespaces are a bit inconsistent, but I could get it working with these in my local composer.json

```json
"psr-4": {
            "PhpTek\\Sentry\\": "vendor/phptek/sentry/src/",
            "PhpTek\\Sentry\\Monolog\\": "vendor/phptek/sentry/src/"
        },
```

### PR DESCRIPTION
**Pull request by tractorcow**
Fix minor namespace bugs / style issues
Fix indentation spaces
Namespace tests
Fixes #50

### CODE REVIEW

File: src/Log/SentryLogger.php [hunk 1]
```diff
@@ -13,9 +13,9 @@
 use SilverStripe\Control\Middleware\TrustedProxyMiddleware;
 use SilverStripe\Core\Injector\Injector;
 use SilverStripe\Dev\Backtrace;
+use SilverStripe\Security\Member;
 use SilverStripe\Security\Security;
 use SilverStripe\Core\Config\Configurable;
-use PhpTek\Sentry\Log\SentryLogger;
```
Comment (phptek) [inline src/Log/SentryLogger.php:18]: Without first importing `SentryLogger` how is `SentryLogger:factory()` supposed to declare `SentryLogger` as a return type?

File: src/Handler/SentryHandler.php [hunk 2]
```diff
@@ -12,12 +12,12 @@
 use Monolog\Handler\AbstractProcessingHandler;
 use Monolog\Logger;
 use Sentry\Severity;
-use Sentry\State\Scope;
 use SilverStripe\Core\Config\Configurable;
 use SilverStripe\Core\Injector\Injectable;
 use PhpTek\Sentry\Log\SentryLogger;
 use PhpTek\Sentry\Adaptor\SentryAdaptor;
 use PhpTek\Sentry\Adaptor\SentrySeverity;
+use Throwable;
```
Comment (phptek) [inline src/Handler/SentryHandler.php:20]: I'm trying to stick to a certain import order (granted, it isn't being adhered-to here)

1. <PHP>
2. <SilverStripe>
3. <3rdParty>
4. <Module>

If you feel like it therefore, `Throwable` should be imported first :-)

File: src/Handler/SentryHandler.php [hunk 3]
```diff
@@ -86,7 +86,7 @@ protected function write(array $record) : void
 
         if (
                 isset($record['context']['exception']) &&
-                $record['context']['exception'] instanceof \Throwable
+                $record['context']['exception'] instanceof Throwable
```
Comment (phptek) [inline src/Handler/SentryHandler.php:89]: I have this feeling that "arguments" passed to `instanceof` needn't be FQCN...(I've always wondered, if that's true, how are class-paths resolved...*ponder*...)

**Comment by phptek** [comment]
@tractorcow thanks for the work :-) - have a squiz my inline comments above. Cheers!

File: src/Log/SentryLogger.php (hunk 1 above)
Comment (tractorcow) [reply src/Log/SentryLogger.php:18]: Classes don't import themselves. It's treated as an error.

File: src/Handler/SentryHandler.php (hunk 2 above)
Comment (tractorcow) [reply src/Handler/SentryHandler.php:20]: My IDE just alphabetises them lol. :) I can arrange.

File: src/Handler/SentryHandler.php (hunk 3 above)
Comment (tractorcow) [reply src/Handler/SentryHandler.php:89]: Sorry I just ran an auto-cleanup tool over my code (does things like prefer imports over FQN).

File: src/Handler/SentryHandler.php (hunk 2 above)
Comment (tractorcow) [reply src/Handler/SentryHandler.php]: Fixed.

File: src/Handler/SentryHandler.php (hunk 3 above)
Comment (tractorcow) [reply src/Handler/SentryHandler.php:89]: As I recall, `instanceof` doesn't trigger autoloading, but using the reference with/without import is purely a stylistic choice and has no real difference in behaviour. :)

As long as you either use the `\` prefix, or you import it (but not both or neither) it's fine.

File: src/Log/SentryLogger.php (hunk 1 above)
Comment (dcentrica) [reply src/Log/SentryLogger.php:18]: OK, I assume you've run it though? (Forgive me, I'm doing more Python atm and my PHP is getting rusty)

Comment (tractorcow) [reply src/Log/SentryLogger.php:18]: Yeah I'm also doing python more these days too. It's always a hassle going back to PHP (never the other way though haha).

**Comment by tractorcow** [comment]
@phptek is this PR acceptable now? :)

**Pull request merged**